website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -255,6 +255,10 @@ of several sub-resources - these resources are laid out below.
     deleting it when destroying the resource through Terraform. If this is set,
     the distribution needs to be deleted manually afterwards. Default: `false`.
 
+  * `wait_for_deployment` (Optional) - By default, the resource will wait
+    for creation and updates to fully deploy. Setting this to `false`
+    will skip waiting for deployments to complete. Default: `true`.
```

Comment:
@bflad I would suggest this to read instead so it's clear what's happening here:

If enabled, the resource will wait for the distribution status to change from `InProgress` to `Deployed`.
Setting this to `false` will skip the process. Default: `true`.

Reply:
👍 great suggestion, will update.